src/citrine/informatics/design_spaces.py

```diff
@@ -60,6 +60,7 @@ class ProductDesignSpace(Resource['ProductDesignSpace'], DesignSpace):
         serializable=False
     )
     active = properties.Boolean('active', default=True)
+    archived = properties.Boolean('archived', default=True)
```

Comment:
Why is the default True for design spaces?

Reply:
woops, that's copy pasta